docs/reference/docs/update.asciidoc

```diff
@@ -35,6 +35,8 @@ curl -XPOST 'localhost:9200/test/type1/1/_update' -d '{
 }'
 --------------------------------------------------
 
+== Append existing array
```

Comment:
Wondering if you should not use `=== Append existing array`

As it's part on the update section?